_layouts/default.html

```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
-    <meta charset='utf-8'>
+    <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Chivo:900">
```

Comment:
This could be tweaked later to include `display`.

https://addyosmani.com/blog/google-fonts-font-display/